### ISSUE TITLE
Tutorial: Fluent + X

### ISSUE THREAD

**Issue by BenjaminRodenberg** [precice/fluent-adapter#2]
Here is a list of possible ideas:

# Lid Driven Cavity with flexible bottom

One possible approach for modelling this would be a simple Bernoulli Beam for the bottom (inhouse python?) + Lid Driven Cavity computed in Fluent.

See [this report, section 2.4.2](https://mediatum.ub.tum.de/doc/1462606/1462606.pdf) for details.

# Rigid Body in Flow (or similar)

Positive side effect: We can use `structure0815` and resolve issue https://github.com/precice/precice/issues/79 at the same time

**Comment by IshaanDesai** [comment]
Currently, the case of a Lid-driven Cavity with a flexible bottom is being pursued and the adapter is being modified accordingly for the FSI case on this branch: https://github.com/precice/fluent-adapter/tree/update_fluent_19_5_R3.
The adapter supports other simulations than FSI but as a first step towards a tutorial, FSI with Lid-dirven Cavity case is being pursued.

**Comment by uekerman** [comment]
You could use the [MBDyn tutorial](https://github.com/precice/mbdyn-adapter/tree/master/tutorials/cavityFSI) as a coupling partner.

**Comment by IshaanDesai** [comment]
> You could use the [MBDyn tutorial](https://github.com/precice/mbdyn-adapter/tree/master/tutorials/cavityFSI) as a coupling partner.

Good idea! Just went through the MBDyn tutorial and I can basically replace OpenFOAM with Fluent and try to use the Structure part as it is! Will update on this shortly :+1:

**Comment by BenjaminRodenberg** [comment]
I see this as a minimal requirement, because without a working tutorial, we cannot make sure that the adapter works.
